### DISCUSSION TITLE
Leading: change to em in place of rem

### DISCUSSION THREAD

**Discussion by cwray-tech** [tailwindlabs/tailwindcss#7666] [Ideas]
In my opinion, [line height styles](https://tailwindcss.com/docs/line-height) of leading-1, leading-2, etc should be based on the text of the line they are styling, not on the base text, which is what rem does. Wondering what made the decision to use rem in those classes instead of em? 

**https://tailwindcss.com/docs/line-height**

**Comment by adamwathan** [comment]
Hey! You will probably enjoy this email update we sent out a few years ago when we first added the fixed line-height values, explains the need for them:

https://ckarchive.com/b/d0ueh0h8kl82

Personally those are all I ever use on projects now. We still have relative values like `loose` for example that behave the way `em` based line-heights would behave, but the fixed value ones are invaluable when trying to make sure everything on the page is sized according to a sensible sizing scale.

**Reply by cwray-tech** [reply]
Hey @adamwathan thank you.. So another question along that lines: when you change font size based on screen size, do you also need to update the line height as well since it is a fixed value? Also, what about headings and text that are text-5xl?
